### PR TITLE
Add documentation for the `/edit-suggestion` command.

### PR DESCRIPTION
**Pull request by Southpaw1496**
There was a suggestion for this on Discord, so here it is.

### CODE REVIEW

**Review by Akarys42 — APPROVED** [review]

**Comment by gdude2002** [comment]
Build failed.

> MaxLineLength - [THREAD_INTRO] at .../src/main/kotlin/org/quiltmc/community/modes/quilt/extensions/suggestions/SuggestionsExtension.kt:64:9

File: src/main/kotlin/org/quiltmc/community/modes/quilt/extensions/suggestions/SuggestionsExtension.kt [hunk 1]
```diff
@@ -61,7 +61,8 @@ private const val ACTION_UP = "up"
 private const val THREAD_INTRO = "This message is at the top of the thread.\n\n" +
         "If this is your suggestion, **please** use `/thread rename` to change the " +
         "name of the thread! You're also welcome to use the other `/thread` commands to manage " +
-        "your suggestion thread as needed."
+        "your suggestion thread as needed. You can edit your suggestion at any time using the `/edit-suggestion`"" +
+" command."
```
Comment (gdude2002) [inline src/main/kotlin/org/quiltmc/community/modes/quilt/extensions/suggestions/SuggestionsExtension.kt]: Still failing - you didn't indent here

Comment (gdude2002) [reply src/main/kotlin/org/quiltmc/community/modes/quilt/extensions/suggestions/SuggestionsExtension.kt]: Also you've got two double quotes at the end of the line before.

**Comment by gdude2002** [comment]
Maybe address both comments before pushing haha, this doesn't compile

**Comment by gdude2002** [comment]
🚀 

Got there in the end - thanks!

**Pull request merged**